tests/generate/cargo_workspace/out/knope.toml

```diff
@@ -32,3 +32,14 @@ name = "document-change"
 
 [[workflows.steps]]
 type = "CreateChangeFile"
+
+[[workflows]]
+name = "get-version"
+help_text = "Get the current version of the project"
+
+[[workflows.steps]]
+type = "Command"
+command = "echo \"$version\""
+
+[workflows.steps.variables]
+"$version" = "Version"
```

Comment:
**issue:** Although this generates successfully for multi-package projects, using the command fails:

```
Error:   × Problem with workflow get-version

Error: variables::too_many_packages

  × Too many packages defined
  help: The Version variable in a Command step can only be used with a single [package].
```

Reply:
Should we scrap this idea completely, or is there a fallback you can suggest?

I guess the primary issue is that the Version variable isn't supported in multi package projects and so we either need a way to detect that, or print the version of each package which is probably not a great idea

Reply:
I only found this repo today, so I'm probably not the best authority for that.

_(brain dump):_

But from my naive perspective, I think a `Version` parameter which mashes together all the versions might make sense. Possibilities are many, it could be a markdown table, a plain list, a grouped list of similar versions. For example, I could imagine a format that looks like this:

```
0.0.1:
    - some-package
    - another-package

0.0.2:
    -a-third-package
    ... etc ...
```

I could also imagine a format that outputs JSON for machine-readable output of package versions. Consider the possibility of feeding JSON into a deployment tool post-release for the new versions we just bumped, etc.

I think it might also make sense for `Version` to use dot-notation to reference the version numbers of individual packages. For example, `Version.some-package` would contain the version number for just `some-package`, etc.

In any case, I'm not sure of the maintainers vision, and whether these ideas align with their own 😄

Reply:
The solution for now is to only add this workflow when there is only a single package defined and omit for multi-package setups. We do need to figure out how to expose versions for monorepos eventually, we should add ideas to #988.